app/assets/stylesheets/application.scss

```diff
@@ -2,9 +2,16 @@
 
 body {
     text-align: center;
-    height: 100vh;
+    margin-bottom: 10vh;
 }
 
+.title {
+    margin: 1vh 2vw 1vh 2vw;
+}
+
+
+// .................................................. Header
```

Comment:
You should try making different scss files to logically chunk your styles for easier management. Then they can be imported into your application.scss file